src/core/platform.js

```diff
@@ -88,76 +70,84 @@ const platform = {
     browser: environment === 'browser',
 
     /**
-     * Is it a desktop or laptop device.
+     * True if running on a desktop or laptop device.
      *
      * @type {boolean}
      */
     desktop: desktop,
 
     /**
-     * Is it a mobile or tablet device.
+     * True if running on a mobile or tablet device.
      *
      * @type {boolean}
      */
     mobile: mobile,
 
     /**
-     * If it is iOS.
+     * True if running on an iOS device.
      *
      * @type {boolean}
      */
     ios: ios,
 
     /**
-     * If it is Android.
+     * True if running on an Android device.
      *
      * @type {boolean}
      */
     android: android,
 
     /**
-     * If it is Windows.
+     * True if running on a Windows mobile device.
```

Comment:
Windows Phone was discontinued 7 years ago:

https://en.wikipedia.org/wiki/Windows_Phone

I would remove this property.

Reply:
done!